filecache/file_cache_source.py

```
@@ -249,20 +250,24 @@ def _is_dir_object_parallel(self,
     @abstractmethod
     def retrieve(self,
                  sub_path: str,
-                 local_path: str | Path) -> Path:
+                 local_path: str | Path,
+                 preserve_mtime: bool = False) -> Path:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)